hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/CreateTableProcedure.java

```diff
@@ -236,8 +236,8 @@ protected void deserializeStateData(ProcedureStateSerializer serializer)
 
   @Override
   protected boolean waitInitialized(MasterProcedureEnv env) {
-    if (getTableName().isSystemTable()) {
-      // Creating system table is part of the initialization, so do not wait here.
+    if (getTableName().equals(TableName.NAMESPACE_TABLE_NAME)) {
```

Comment:
There are no NAMESPACE_TABLE now?

Reply:
For keeping the code same for both master and branch-2 for now, so it will be easy to cherry-pick to branch-2. Could open a follow on issue to remove it on master? Or I could open another PR for branch-2 after the master one is merged and added this check.

Reply:
OK, checked the code, on branch-2 we still do not need to create namespace before loading meta. The creation of namespace is in initClusterSchemaService, which is far after we finish loading meta...